pkg/planner/core/casetest/index/testdata/integration_suite_in.json

```diff
@@ -19,5 +19,18 @@
       "select b from t3 where a = 1 and b is not null",
       "select b from t3 where a = 1 and b is null"
     ]
+  },
+  {
+    "name": "TestRowFunctionMatchTheIndexRangeScan",
```

Comment:
Looks good overall. Could you add tests: (1) between like (k1,k2) between ...  (2) mix of upper and lower bounds (3) row function and ranges on individual columns.